packages/cli/native_modules.rb

```diff
@@ -0,0 +1,219 @@
+def use_native_modules!(packages = nil)
+  if (!packages)
+    cli_bin = File.expand_path("../build/index.js", __FILE__)
+    output = Pod::Executable.execute_command("node", [cli_bin, "config"], true)
+    config = JSON.parse(output)
+    packages = config["dependencies"]
+  end
+
+  found_pods = []
+
+  packages.each do |package_name, package|
+    next unless package["ios"]
+
+    podspec_path = File.join(package["root"], package["ios"]["podspec"])
+    spec = Pod::Specification.from_file(podspec_path)
+
+    # We want to do a look up inside the current CocoaPods target
+    # to see if it's already included, this:
+    #   1. Gives you the chance to define it beforehand
+    #   2. Ensures CocoaPods won't explode if it's included twice
+    #
+    this_target = current_target_definition
+    existing_deps = current_target_definition.dependencies
+
+    # Skip dependencies that the user already activated themselves.
+    next if existing_deps.find do |existing_dep|
+      existing_dep.name.split('/').first == spec.name
+    end
+
+    pod spec.name, :path => package["root"]
+
+    if package["ios"]["scriptPhases"]
+      # Can be either an object, or an array of objects
+      Array(package["ios"]["scriptPhases"]).each do |phase|
+        # see https://www.rubydoc.info/gems/cocoapods-core/Pod/Podfile/DSL#script_phase-instance_method
+        # for the full object keys
+
+        # Support passing in a path instead of the raw script
+        if phase["path"]
+          phase["script"] = File.read(phase["path"])
```

Comment:
A path would be relative to the node_modules root of the package, e.g. `./some_script.sh` rather than being fully qualified. But we have the root at `package["root"]` so should be ok to change this I think and use a FQ path?

Reply:
Yep, you're right 👍

Reply:
@Salakar Done, thanks!

Reply:
@orta @alloy awesome, thanks!